code/src/components/AllTodoItems.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { todoList } from 'reducers/todoList'
+import styled from 'styled-components'
+
+const AllTodoItems = () => {
+  const todoItems = useSelector((store) => {
+    return store.todolist.list;
+  });
+  const dispatch = useDispatch()
+
+  const handleCheck = ({ todoId }) => {
+    dispatch(todoList.actions.handleChecked({ todoId }))
+  }
+
+  const handleDelete = ({ todoId }) => {
+    dispatch(todoList.actions.handleDeleted({ todoId }))
+  }
+
+  return (
+    <CheckboxContainer>
+      {todoItems.length > 0 ? (
+      <> 
+      {todoItems.map((todo) => (
+        <CheckboxItem key={todo.id}>
+          <CheckboxDisplay>
+            <Input
+              id={todo.id}
+              type="checkbox"
+              name={todo.task}
+              checked={todo.isCompleted}
+              onChange={() => handleCheck({ todoId: todo.id })}
+            />
+            <Label htmlFor={todo.id}>
+              <span>{todo.task}</span>
+            </Label>
+          </CheckboxDisplay>
+          <Button
+            onClick={()=> handleDelete({ todoId: todo.id })}>
+            {/* {todo.delete} */}
+              🗑
+          </Button>
+        </CheckboxItem>
+      ))}
+      </>
+      ) : (<p>You have no tasks left today</p>)}
+    </CheckboxContainer>
+  );
+}
+export default AllTodoItems
+
+const Input = styled.input`
+  display: inline-block;
+  }
+`
+const CheckboxContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+  `
```

Comment:
Nice that you're continuing with styled components!
The closing backtick should not be indented. Change on multiple places

Reply:
Thanks for pointing this out. All is fixed accordingly.